lib/internal/bootstrap_node.js

```diff
@@ -74,6 +68,12 @@
 
     _process.setupRawDebug();
 
+    const browserGlobals = !process._noBrowserGlobals;
+    if (browserGlobals) {
+      setupGlobalTimeouts();
```

Comment:
Suggestion:
```js
// This needs to happen after `_process.setupChannel()`
```

Reply:
There's a regression test and `git blame`, so I don't think it's necessary (or even useful in this form tbh).